src/main/osd/osd_elements.c

```diff
@@ -1734,6 +1734,18 @@ void osdUpdateAlarms(void)
         CLR_BLINK(OSD_ALTITUDE);
     }
 
+#ifdef USE_GPS
+    if (sensors(SENSOR_GPS) && ARMING_FLAG(ARMED) && STATE(GPS_FIX) && STATE(GPS_FIX_HOME)) {
```

Comment:
In this case I'd keep the blinking on even when disarmed - the distance will still be over. :wink:

Reply:
Well it's a little complicated... Under the default settings the the

Reply:
S all good - I just thought keeping it blinking while disarmed was a quick win. But ultimately I don't think it makes much of a difference.